scanner/src/main/kotlin/scanners/scancode/ScanCodeResultParser.kt

```diff
@@ -257,7 +257,7 @@ private fun getCopyrightFindings(result: JsonNode): List<CopyrightFinding> {
         val copyrights = file["copyrights"]?.asSequence().orEmpty()
         copyrights.map { copyright ->
             CopyrightFinding(
-                statement = copyright["value"].textValue(),
+                statement = (copyright["value"] ?: copyright["copyright"]).textValue(),
```

Comment:
Would it make sense to add a code comment which makes it more obvious that the case distinctions is for < 2.0 and >= 2.0?

Reply:
Would you mind me doing this later so we can merge this fix ASAP?

Reply:
See https://github.com/oss-review-toolkit/ort/pull/6212.